### ISSUE TITLE
Midterm report Review

### ISSUE THREAD

**Issue by yc2575** [medhavi27/Hot-Spotify-Tracks#7]
In this midterm report, the group tries to determine the features that could make songs popular and also predict the popularity of songs using different features. They acquired the data from Spotify Web API and the total number of raw features they have is 13 and most of them are numeric except artist. They also consider to take variable years as categorical, which is the first point I like. Since even year is the number but it does not have the real meaning and every year might represent a new trend, which is more like category of trend rather than a number. Second thing I like is the way they separate the training set and test set. Since in real world, we cannot see the trend of the future and therefore, we always try to predict things from past. If we just split those two sets regularly, that would make the assumption of we have already capture some trend in futures. Last one I like is they have already realize some outliers that exist in the model, such as some features may dominate others. Therefore, they need to analyze it more carefully. 
What I think need some improvement is that since they have already realized that the year is categorical and it only represent the trend. They need to find the way to predict the trend by some other features, since even within a small time, the trend could totally change and make their model useless. Second one is those measurements, from the report I could not tell who give those rates, but there might be a dominate factor that affect all the rates. They need to identify such situation. Lastly, I think they have 10000 songs, but those rating are only from 0 to 1 and have 10 scales. Those feature may not be able to reflect the popularity at a very detailed sense and they might need to include more features.
